go/internal/initutil/ipfs.go

```diff
@@ -37,6 +40,10 @@ func (m *Manager) SetupLocalIPFSFlags(fs *flag.FlagSet) {
 	fs.DurationVar(&m.Node.Protocol.MaxBackoff, "p2p.max-backoff", time.Minute, "maximum p2p backoff duration")
 	fs.BoolVar(&m.Node.Protocol.LocalDiscovery, "p2p.local-discovery", true, "local discovery")
 	fs.Var(&m.Node.Protocol.RdvpMaddrs, "p2p.rdvp", `list of rendezvous point maddr, ":dev:" will add the default devs servers, ":none:" will disable rdvp`)
+	fs.BoolVar(&m.Node.Protocol.Tor.Enabled, "tor.enable", false, "If true tor will be enabled.")
+	fs.StringVar(&m.Node.Protocol.Tor.BinaryPath, "tor.binary-path", "", "If set berty will use this external tor binary instead of his builtin one.")
+	fs.BoolVar(&m.Node.Protocol.Tor.DontSetListen, "tor.dont-set-listen", false, "If true tor will not auto set his listen address, allowing you to replace by an other one.")
+	fs.BoolVar(&m.Node.Protocol.AnonymityMode, "anonymity-force", false, `If true the node will be configurated to only use anonimous transports and will disable local discovery. Implies "-tor.enable=true"`)
```

Comment:
```suggestion
	fs.BoolVar(&m.Node.Protocol.Tor.Enabled, "tor.enable", false, "if true tor will be enabled")
	fs.StringVar(&m.Node.Protocol.Tor.BinaryPath, "tor.binary-path", "", "if set berty will use this external tor binary instead of his builtin one")
	fs.BoolVar(&m.Node.Protocol.Tor.DontSetListen, "tor.dont-set-listen", false, "if true tor will not auto set his listen address, allowing you to replace by an other one")
	fs.BoolVar(&m.Node.Protocol.AnonymityMode, "anonymity-force", false, `if true the node will be configurated to only use anonimous transports and will disable local discovery. Implies "-tor.enable=true"`)
```

be consistent with the lines just above